scripts/install

```diff
@@ -101,41 +107,28 @@ phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
 #=================================================
 ynh_script_progression --message="Configuring Kanboard..."
 
-# Retrieve admin email
-email=$(ynh_user_get_info --username=$admin --key=mail)
+cp ../conf/config.php "$final_path/config.php"
 
-# Copy and edit config.php
-config_php="${final_path}/config.php"
+ynh_replace_string --match_string="__DB_PWD__" --replace_string="$db_pwd" --target_file="$final_path/config.php"
+ynh_replace_string --match_string="__DB_NAME__" --replace_string=$db_name --target_file="$final_path/config.php"
+ynh_replace_string --match_string="__ADMIN__" --replace_string=$admin --target_file="$final_path/config.php"
+ynh_replace_string --match_string="__EMAIL__" --replace_string=$email --target_file="$final_path/config.php"
+ynh_replace_string --match_string="__DOMAIN__" --replace_string=$domain --target_file="$final_path/config.php"
```

Comment:
Would be cool to use ynh_add_config instead ^^

Reply:
Ideally yes but I don't know how to keep this upstream `__DIR__`variable from being replaced by the `ynh_add_config` helper...
https://github.com/YunoHost-Apps/kanboard_ynh/blob/e724e2e8c5477c2dc71c44e61766df3a92be298f/conf/config.php#L4

Reply:
don't know either maybe having `dir="__DIR__"` ... not sur it will work. but you can merge without `ynh_ad__config`